app/api/models/errors/SeBusinessCommonErrors.scala

```diff
@@ -90,3 +90,38 @@ object RuleWrongTpaAmountSubmittedError
       code = "RULE_WRONG_TPA_AMOUNT_SUBMITTED",
       message = "Transition profit acceleration value cannot be submitted without a transition profit value",
       BAD_REQUEST)
+
```

Comment:
Error messages dont match the error messages in the business requirements, check them and correct

Reply:
this is an existing code and not part of the PR